.github/workflows/newman.yml

```diff
@@ -41,8 +48,179 @@ jobs:
           echo "GITHUB_OUTPUT contents:"
           cat $GITHUB_OUTPUT
 
+  associate-test-cases:

```

Comment:
as discussed, this workflow should be run in following environments : `unstable` , `testing` , `stable`